simulate_presence_script.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+
+import requests
+import json
+import random
+seat_seat = 'http://127.0.0.1:8000/api/seats/'
+halls_url = 'http://127.0.0.1:8000/api/halls/'
```

Comment:
Since the URL base repeats three times, I think this calls for a new variable to store it.
You can even make a method that uses urljoin to return the full URL to be used in the calls.